.github/workflows/build.yml

```diff
@@ -1047,7 +1047,7 @@ jobs:
           key: android-${{ matrix.os }}-cmake-${{ matrix.EVENT_MATRIX }}-v4
 
       - name: Set up JDK 17
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v4
```

Comment:
Interesting, it seems that this breaks android builds - #1716, @theoden8 maybe you will be interested

```
/home/runner/work/libevent/libevent/evutil.c:3079:6: error: call to undeclared function 'pipe2'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
 3079 |         if (pipe2(fd, O_NONBLOCK|O_CLOEXEC) == 0)
      |             ^
/home/runner/work/libevent/libevent/evutil.c:3079:6: note: did you mean 'pipe'?
/usr/local/lib/android/sdk/ndk/27.1.1229[70](https://github.com/libevent/libevent/actions/runs/11127592141/job/30920267436?pr=1724#step:7:71)06/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/unistd.h:275:5: note: 'pipe' declared here
  275 | int pipe(int __fds[_Nonnull 2]);
      |     ^
1 error generated.
```

Reply:
Never mind, it should be fixed by https://github.com/libevent/libevent/pull/1721